tests/testthat/test-carehomes-multistrain.R

```diff
@@ -764,3 +764,241 @@ test_that("calculate Rt with both second variant and vaccination", {
   }
 
 })
+
+test_that("strain_rel_gamma works as expected in carehomes_parameters", {
+  expect_silent(carehomes_parameters(sircovid_date("2020-02-07"), "england",
+                                     strain_rel_gamma_A = 1,
+                                     strain_rel_gamma_P = 1,
+                                     strain_rel_gamma_C_1 = 1,
+                                     strain_rel_gamma_C_2 = 1))
+  expect_silent(carehomes_parameters(sircovid_date("2020-02-07"), "england",
+                                     strain_rel_gamma_A = 1,
```

Comment:
you are going to need to change those tests if you follow my suggestion of forcing the first value in each of those to be 1 (so that the 1st strain is the reference)